spec/lib/danger/helpers/comments_helper_spec.rb

```diff
@@ -267,34 +267,7 @@ class Dummy
 
       expect(comment).to include(summary)
     end
-  end
-
-  describe "#generate_description" do
-    it "Handles no errors or warnings" do
-      message = dummy.generate_description(warnings: [], errors: [])
-      expect(message).to include("All green.")
-    end
-
-    it "handles a single error and a single warning" do
-      message = dummy.generate_description(warnings: [1], errors: [1])
-
-      expect(message).to include("⚠️ ")
-      expect(message).to include("Error")
-      expect(message).to include("Warning")
-      expect(message).to include("Don't worry, everything is fixable.")
-    end
-
-    it "handles multiple errors and warning with pluralisation" do
-      message = dummy.generate_description(warnings: [1, 2], errors: [1, 2])
-
-      expect(message).to include("⚠️ ")
-      expect(message).to include("Errors")
-      expect(message).to include("Warnings")
-      expect(message).to include("Don't worry, everything is fixable.")
-    end
-  end
 
-  describe "#generate_comment" do
```

Comment:
Thats the meat of the diff in this file lol. I merged two `#generate_comment` blocks, the diff format just makes it look funky.